<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RBNomenclatureActiveSubstanceEditor</class>
 <widget class="QDialog" name="RBNomenclatureActiveSubstanceEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>632</width>
    <height>423</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ChangeMe!</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="margin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabMain">
      <attribute name="title">
       <string>Описание</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <property name="margin">
        <number>4</number>
       </property>
       <property name="spacing">
        <number>4</number>
       </property>
       <item row="0" column="1">
        <widget class="QLineEdit" name="edtCode"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="lblATC">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Код &amp;АТХ</string>
         </property>
         <property name="buddy">
          <cstring>edtATC</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="edtNameLatin"/>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="edtATC"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lblNameLatin">
         <property name="text">
          <string>Наименование на латыни</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="lbDosageUnit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Ед. Дозировки</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lblNameRussian">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Наименование на русском</string>
         </property>
         <property name="buddy">
          <cstring>edtNameRussian</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="lblCode">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>&amp;Код</string>
         </property>
         <property name="buddy">
          <cstring>edtCode</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="edtNameRussian"/>
       </item>
       <item row="4" column="1">
        <widget class="CRBComboBox" name="cmbUnit"/>
       </item>
       <item row="5" column="0" colspan="2">
        <widget class="CInDocTableView" name="tblActiveSubstanceGroups"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabIdentification">
      <attribute name="title">
       <string>Идентификация</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="margin">
        <number>4</number>
       </property>
       <property name="spacing">
        <number>4</number>
       </property>
       <item row="0" column="0">
        <widget class="CInDocTableView" name="tblIdentification"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CRBComboBox</class>
   <extends>QComboBox</extends>
   <header>library/crbcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>CInDocTableView</class>
   <extends>QTableView</extends>
   <header>library/InDocTable.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>edtCode</tabstop>
  <tabstop>edtNameRussian</tabstop>
  <tabstop>edtNameLatin</tabstop>
  <tabstop>edtATC</tabstop>
  <tabstop>cmbUnit</tabstop>
  <tabstop>tblActiveSubstanceGroups</tabstop>
  <tabstop>buttonBox</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>tblIdentification</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>RBNomenclatureActiveSubstanceEditor</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>154</x>
     <y>101</y>
    </hint>
    <hint type="destinationlabel">
     <x>154</x>
     <y>58</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>RBNomenclatureActiveSubstanceEditor</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>154</x>
     <y>101</y>
    </hint>
    <hint type="destinationlabel">
     <x>154</x>
     <y>58</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
